addons/hr_holidays/tests/test_out_of_office.py

```diff
@@ -92,11 +92,15 @@ def test_leave_im_status_performance_partner_offline(self):
     @users('__system__', 'demo')
     @warmup
     def test_leave_im_status_performance_user_leave_offline(self):
+        while self.leave.state != 'validate':
```

Comment:
Why is the while needed here? :thinking:

Reply:
I think it's because I had some trouble with leaves that need double approbation, so I added this to ensure I have the correct state.

Reply:
In that case you can just write the state you wish, it would be easier

Reply:
It's fixed! :)